x/gov/README.md

```diff
@@ -268,6 +268,40 @@ Validators and full nodes can use an automation tool, such as [Cosmovisor](https
 
 ## State
 
+### Constitution
```

Comment:
Maybe we should set guideline on how is formatted a constitution.
If chains use an IPFS link or set it directly in the field, it might be annoying for explorers to parse and display it.

Reply:
great idea, I am hoping for Markdown stored directly in genesis, to enhance the durability of the document.